packages/lesswrong/server/callbacks/commentCallbacks.ts

```diff
@@ -416,3 +416,15 @@ async function updateTopLevelCommentLastCommentedAt (comment) {
 addCallback("comment.create.after", updateTopLevelCommentLastCommentedAt)
 
 addCallback("comment.create.after", newDocumentMaybeTriggerReview)
+
+
+async function updateLastCommentPromotedAt (comment, {oldDocument}) {
```

Comment:
I don't think you should do this with a callback, but instead use the `onInsert` and `onUpdate` fields in the schema, which automatically register callbacks.